src/Resources/ResourceManager/Implementation/Deployments/NewAzureManagementGroupDeploymentCmdlet.cs

```diff
@@ -65,6 +65,8 @@ public class NewAzureManagementGroupDeploymentCmdlet : DeploymentCreateCmdlet
         [ValidateChangeTypes]
         public string[] WhatIfExcludeChangeType { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "The query string (for example, a SAS token) to be used with the templateLink URI. Would be used in case of linkded templates")]
```

Comment:
I believe you were refering to `-TemplateUri` because there's no templateLink term in PowerShell.
```suggestion
        [Parameter(Mandatory = false, HelpMessage = "The query string (for example, a SAS token) to be used with the TemplateUri parameter. Would be used in case of linkded templates")]
```